playbooks/adhoc/uninstall.yml

```diff
@@ -574,3 +597,19 @@
     # - /etc/systemd/system/haproxy.service.d
     - /etc/systemd/system/haproxy.service.d/limits.conf
     - /etc/systemd/system/haproxy.service
+
+# Remove Contrail vRouter
+- hosts: nodes
+  become: yes
+  tasks:
+  - name: Find nodes which has Contrail vRouter
+    when: item == "vhost0"
+    with_items:
+      - "{{ ansible_interfaces | map('replace', '-','_') | list }}"
+    register: vhost_register
+
+  - name: Reboot Contrail vRouter nodes
+    shell: /sbin/shutdown -r +1
```

Comment:
I thought this was all daemonset. shutdown?

Reply:
Reboot the vRouter nodes after cleaning up the vRouter processes

Reply:
why is a reboot needed?

Reply:
Aren't these nodes where there could be other pods running too?